Source/Classes/NantesLabel.swift

```diff
@@ -132,33 +133,33 @@ open class NantesLabel: UILabel {
     }
 
     /// The distance, in points, from the leading margin of a frame to the beginning of the line paragraph's first line. 0.0 by default
-    open var firstLineIndent: CGFloat = 0.0
+    @IBInspectable open var firstLineIndent: CGFloat = 0.0
 
     /// The shadow color for the label when the label's `highlighted` property is true. Defaults to nil
-    open var highlightedShadowColor: UIColor?
+    @IBInspectable open var highlightedShadowColor: UIColor?
 
     /// The shadow offset for the label when the label's `highlighted` propert is true. A value of 0 indicates no blur, larger numbers indicate more blur. This can't be a negative value.
-    open var highlightedShadowOffset: CGSize = .zero
+    @IBInspectable open var highlightedShadowOffset: CGSize = .zero
 
     /// The shadow blur radius for the label. A value of 0 indicates no blur. Larger values indicate more blur. This can't be a negative value.
-    open var highlightedShadowRadius: CGFloat = 0.0
+    @IBInspectable open var highlightedShadowRadius: CGFloat = 0.0
 
     /// NSAttributedString attributes used to style inactive links
     /// nil or [:] will add no styling
-    open var inactiveLinkAttributes: [NSAttributedString.Key: Any]?
+    @IBInspectable open var inactiveLinkAttributes: [NSAttributedString.Key: Any]?
 
     /// Floating point number in points; amount to modify default kerning. 0 means kerning is disabled. nil uses default kerning
```

Comment:
Can you reword this comment now that `kern` can't be nil?
Maybe something like:
`/// Floating point number in points; amount to modify default kerning. 0 means kerning is disabled. 0 is the default.`